docs/index.asciidoc

```diff
@@ -58,6 +72,19 @@ input plugins.
 
 SNMP Community String to listen for.
 
+[id="plugins-{type}s-{plugin}-ecs_compatibility"]
+===== `ecs_compatibility`
+
+  * Value type is <<string,string>>
+  * Supported values are:
+    ** `disabled`: does not use ECS-compatible field names (fields might be set at the root of the event)
+    ** `v1`, `v8`: avoids field names that might conflict with Elastic Common Schema (for example, JMS specific properties)
```

Comment:
```suggestion
    ** `v1`, `v8`: avoids field names that might conflict with Elastic Common Schema (for example, SNMP specific properties)
```

Reply:
Good catch! Thank you

Reply:
updated to https://github.com/logstash-plugins/logstash-input-snmptrap/pull/37/commits/faad777e738818895005e951680bc5fc48529fcc